components/Dropdown/Dropdown.js

```diff
@@ -1,25 +1,24 @@
 class Dropdown {
   constructor(element) {
-    
-    // Assign this.element to the dropdown element
-    this.element;
-    
+    this.element = element;
     // Get the element with the ".dropdown-button" class found in the dropdown element (look at the HTML for context)
-    this.button = this.element.querySelector();
-    
+    this.button = this.element.querySelector('.dropdown-button');
     // assign the reference to the ".dropdown-content" class found in the dropdown element
-    this.content;
-    
+    this.content = this.element.querySelector('.dropdown-content');
     // Add a click handler to the button reference and call the toggleContent method.
     this.button.addEventListener('click', () => {
-
+      this.toggleContent();
     })
   }
 
   toggleContent() {
-    
     // Toggle the ".dropdown-hidden" class off and on
-    this.content;
+    const DROPDOWN_HIDDEN_CLASS = 'dropdown-hidden'
```

Comment:
This is a great habit to get into. Nice use of a const to avoid spelling errors and use code editors autocomplete